infrastructure/network/netadapter/server/grpcserver/protowire/p2p.proto

```diff
@@ -161,7 +161,7 @@ message OutpointAndUtxoEntryPair{
 message UtxoEntry {
   uint64 amount = 1;
   ScriptPublicKey scriptPublicKey = 2;
-  uint64 blockBlueScore = 3;
+  uint64 blockDaaScore = 3;
```

Comment:
You have places where you spell Daa and places where you spell DAA, I think all should be DAA

Reply:
AFAIK the convention in proto files is to use js-like camel case